api/spec/requests/ransackable_attributes_spec.rb

```diff
@@ -33,7 +33,7 @@
 
       get "/api/variants?q[product_name_or_sku_cont]=fritos", token: user.spree_api_key
 
-      skus = JSON.parse(response.body)['variants'].map { |variant| variant['sku'] }
+      skus = JSON.parse(response.body)['variants'].map { |x| x['sku'] }
```

Comment:
Why is this beneficial? I'd find `variant` more descriptive as a variable name than `x`.